app/javascript/lib/validation/validate-file.js

```diff
@@ -61,7 +61,7 @@ async function validateLocalFile(file, studyFile, allStudyFiles=[], allowedFileE
 
   let issuesObj
 
-  const noContentValidationFileTypes = ['Seurat', 'AnnData', 'Other', 'Documentation']
+  const noContentValidationFileTypes = ['Seurat', 'AnnData Reference', 'AnnData Ingestible', 'Other', 'Documentation']
```

Comment:
Blocker: per other comment:

```suggestion
  const noContentValidationFileTypes = ['Seurat', 'AnnData Reference', 'AnnData', 'Other', 'Documentation']
```